### PR TITLE
Vc update

### PR DESCRIPTION
**Pull request by xwzheng1020**
add design of vc_update

### CODE REVIEW

**Comment by Gerhut** [comment]
Should these designs be put into webportal parts?

File: docs/rest-server/design-of-vc-update.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+（1）点击“Add“ 按钮，弹出对话框，在其中编辑拟添加vc的名称和容量值。支持在一个页面内填写多个vc。
+
+（2）在弹出的对话框中，点击“Add”按钮，逐条添加VC。每次添加VC调用一次接口` PUT /api/v1/virtual-clusters/:vcName`，若更新成功则显示“succeed”，更新失败，醒目地显示失败原因。单次调用失败不影响后续过程。
+
+### 3.2 delete a vc（admin only）
+
+![1545897742293](./delete_vc.png)
+
+（1）点击“Delete”按钮，弹出提示框“是否确认要删除:vcName"，若否，则不做变更；若是，调用`DELETE /api/v1/virtual-clusters/:vcName`，并弹出消息框显示返回信息
```
Comment (mzmssg) [inline docs/rest-server/design-of-vc-update.md]: 这里可以添加提示让admin先停止VC下的job，不然delete会失败。如果需要做job的decommission，可以把vc status置为stopped，等到job都完成了再delete

**Comment by fanyangCS** [comment]
please change the text to English.

**Comment by xwzheng1020** [comment]
@fanyangCS 
The text has been changed to English. 
The figures have been updated.

File: docs/rest-server/design-of-vc-update.md (hunk 1 above)
Comment (xwzheng1020) [reply docs/rest-server/design-of-vc-update.md]: A note has been added to delete confirm box. "If there are jobs of this virtual cluster still running, it cannot be deleted."

Comment (xwzheng1020) [reply docs/rest-server/design-of-vc-update.md]: A design can be found [here](https://github.com/Leinao/pai/blob/vc_update/docs/webportal/delete_vc.png)

**Comment by fanyangCS** [comment]
is this PR obsoleted? if so, please close it.

**Comment by fanyangCS** [comment]
shall we close this PR?

**Comment by mzmssg** [comment]
Closed

**Pull request closed without merging**